tests/gumjs/script.c

```diff
@@ -3345,6 +3351,54 @@ TESTCASE (file_can_be_written_to)
   g_free (contents);
 }
 
+TESTCASE (file_apis_can_not_trigger_interceptor)
```

Comment:
This needs a corresponding `#ifndef` I think to avoid warnings about unused functions.